Packs/FortiSIEM/Integrations/FortiSIEM/FortiSIEM.yml

```diff
@@ -290,7 +290,7 @@ script:
       description: ID of the resource group. Run the "fortisiem-get-lists" command to get the ID. command.
     - name: object-info
       required: true
-      description: 'CSV list of key-value pairs of attributes, for example: name=SomeName,lowIp=192.168.1.1,highIp=192.168.1.2'
+      description: 'CSV list of key-value pairs of attributes, for example: name=SomeName,lowIp=192.168.1.1,highIp=192.168.1.2.'
```

Comment:
```suggestion
      description: 'CSV list of key-value pairs of attributes (for example: name=SomeName,lowIp=192.168.1.1,highIp=192.168.1.2).'
```

Reply:
@ShirleyDenkberg, what do you think about this?

Reply:
Either way would be OK.